modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5451,6 +5451,7 @@ default-max-connections-per-route=Default Max Connections per Route
 default-page-audit-image-alt-description=Default Page Audit image.
 default-permissions=Default Permissions
 default-permissions-company-configuration-name=Default Permissions Configuration
+default-permissions-group-configuration-name=Default Permissions Configuration
```

Comment:
why you didn't merge this 2 keys?

Reply:
Because are the name of two different configurations (with the same label)